src/components/cells/cell-menu-container.js

```diff
@@ -109,6 +109,8 @@ function mapStateToProps(state, ownProps) {
     label = 'resource'
   } else if (cell.cellType === 'raw') {
     label = 'raw'
+  } else if (cell.cellType === 'fetch') {
+    label = 'fetch'
```

Comment:
Could you simplify/generalize this code by using `cellTypeToJsmdMap` in jsmd-tools? (might be better done in a followup)

Reply:
nice catch will. i think it actually makes sense to do this now -- easier/cleaner/faster than having another issue+PR hanging over us

Reply:
Sure, haven't really touched this so I'll have to look into what you mean :)